src/main/java/com/artipie/Settings.java

```diff
@@ -49,4 +52,43 @@
      * @throws IOException In case of problems with reading settings.
      */
     CompletionStage<Authentication> auth() throws IOException;
+
+    /**
+     * Fake {@link Settings} using a file storage.
+     *
+     * @since 0.2
+     */
+    final class Fake implements Settings {
+
+        /**
+         * Storage path.
+         */
+        private final Path storage;
+
+        /**
+         * Vertx.
+         */
+        private final Vertx vertx;
+
+        /**
+         * Ctor.
+         *
+         * @param storage Storage path
+         * @param vertx Vertx
+         */
+        public Fake(final Path storage, final Vertx vertx) {
+            this.storage = storage;
+            this.vertx = vertx;
+        }
+
+        @Override
+        public Storage storage() throws IOException {
+            return new FileStorage(this.storage, this.vertx.fileSystem());
```

Comment:
@victornoel can we use `InMemoryStorage` for testing? I think it has more sense for fake implementation here and it doesn't depend on FS resources